docs/reference/esql/functions/date-time-functions.asciidoc

```diff
@@ -8,7 +8,7 @@
 {esql} supports these date-time functions:
 
 // tag::date_list[]
-* experimental:[] <<esql-auto_bucket>>
+* <<esql-bucket>>
```

Comment:
Is it intended that we declare `BUCKET` as released in this PR?

(If so, let's add this to the PR description as it's important IMHO.)

Reply:
> Is it intended that we declare BUCKET as released in this PR?

Yes, I believe we wanted to decide on the naming, also considering the non-auto version.